### PR TITLE
DOCSP-18980 improve mongoshrc visibility

### PR DESCRIPTION
**Pull request by davemungo**
https://jira.mongodb.org/browse/DOCSP-18980
Improve visibility of .mongoshrc section

### STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-18980-improve-mongoshrc-visibility/changelog/#v0.13.1

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-18980-improve-mongoshrc-visibility/mongoshrc/#std-label-mongoshrc-js

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-18980-improve-mongoshrc-visibility/write-scripts/#execute-code-from-a-configuration-file

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-18980-improve-mongoshrc-visibility/run-commands/#.mongoshrc.js-file

### CODE REVIEW

File: source/includes/examples/ex-display-dbname-and-hostname.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Use a function like this one to display the database and hostname in
```
Comment (jason-price-mongodb) [inline source/includes/examples/ex-display-dbname-and-hostname.rst:1]: "To display XXX, use YYY"

File: source/includes/examples/ex-display-system-uptime.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Use a function like this one for a prompt that shows the system uptime
```
Comment (jason-price-mongodb) [inline source/includes/examples/ex-display-system-uptime.rst:1]: "To XXX, use YYY"

File: source/includes/fact-mongoshrc-loading.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
+On startup, :binary:`mongosh` checks the user's ``HOME`` directory for
+a JavaScript file named :ref:`.mongoshrc.js <mongoshrc-js>`. If this
+file is found, :binary:`mongosh` interprets the content of
```
Comment (jason-price-mongodb) [inline source/includes/fact-mongoshrc-loading.rst:3]: "reads the content"

File: source/mongoshrc.txt [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+..
+   COMMENT This file might be expanded later. For now the content is
+   same here and in run-commands.
+
+.. include:: /includes/fact-mongoshrc-loading.rst
+
+Examples
+--------
+
+Create custom prompts for :binary:`~bin.mongosh`. Add the code in the
```
Comment (jason-price-mongodb) [inline source/mongoshrc.txt:24]: "To set XXX, YYY"

File: source/write-scripts.txt [hunk 5]
```diff
@@ -434,6 +434,53 @@ The following lines are equivalent.
...
+
+.. code-block:: javascript
+
+   db.clientConnections.updateOne(
+      { "_id": ObjectId() },
+      { $currentDate: { "connectTime": true } },
+      { upsert: true }
+   ) ;
+
+Each time you connect to a database, the MongoDB server will add a
```
Comment (jason-price-mongodb) [inline source/write-scripts.txt:468]: "adds a"

**Review by jason-price-mongodb — APPROVED** [review]
Hey Dave, LGTM with suggestions. Cheers, Jason.

**Review by davemungo — COMMENTED** [review]
Thanks @jason-price-mongodb ,  I'll send this out to external review next

File: source/write-scripts.txt [hunk 6]
```diff
@@ -434,6 +434,53 @@ The following lines are equivalent.
...
+To create a log of when your :binary:`mongosh` client connects to a
+database, add the following to ``<your-home-directory>/.mongoshrc.js``:
+
+.. code-block:: javascript
+
+   db.clientConnections.updateOne(
+      { "_id": ObjectId() },
+      { $currentDate: { "connectTime": true } },
+      { upsert: true }
+   ) ;
```
Comment (addaleax) [inline source/write-scripts.txt:466]: Is there a reason that this isn’t `db.clientConnections.insertOne({ connectTime: ISODate() })`?

**Review by addaleax — APPROVED** [review]

File: source/write-scripts.txt (hunk 6 above)
Comment (davemungo) [reply source/write-scripts.txt]: Thanks for the suggestion, updated.

File: source/write-scripts.txt (hunk 5 above)
Comment (davemungo) [reply source/write-scripts.txt:464]: Done

**Pull request merged**